dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -54,76 +54,76 @@ CAPTION "Securitate"
-    GROUPBOX "Informații de autentificare", IDC_STATIC, 7, 50, 261, 52
-    LTEXT "<Message>", IDC_SECURITY_MESSAGE, 15, 62, 245, 18
-    LTEXT "<Logon date>", IDC_SECURITY_LOGONDATE, 15, 83, 245, 8
-    CTEXT "Ce acțiune preferați?", IDC_STATIC, 10, 108, 255, 8
-    PUSHBUTTON "&Blochează calculatorul", IDC_SECURITY_LOCK, 7, 124, 80, 14
-    PUSHBUTTON "&Deautentifică-mă…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
-    PUSHBUTTON "Închid&e calculatorul…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
-    PUSHBUTTON "Schim&bă parola…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
-    PUSHBUTTON "&Gestionar de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 188, 143, 80, 14
+    GROUPBOX "Informaţii de Log on", IDC_STATIC, 7, 50, 261, 52
+    LTEXT "<Mesaj>", IDC_LOCKED_MESSAGE, 15, 62, 245, 18
+    LTEXT "<Dată Log on:>", IDC_SECURITY_LOGONDATE, 15, 83, 245, 8
+    CTEXT "Ce vreți să faceți?", IDC_STATIC, 10, 108, 255, 8
+    PUSHBUTTON "&Blocare computer", IDC_SECURITY_LOCK, 7, 124, 80, 14
+    PUSHBUTTON "&Log off…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
+    PUSHBUTTON "Închidere…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
+    PUSHBUTTON "M&odificare parolă…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
+    PUSHBUTTON "&Manager de activităţi", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 188, 143, 80, 14
 END
 
 IDD_LOCKED DIALOGEX 0, 0, 275, 121
 STYLE NOT WS_VISIBLE | DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_BORDER | WS_CAPTION | WS_DLGFRAME | WS_POPUP
-CAPTION "Calculator blocat"
+CAPTION "Computer blocat"
 FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
     ICON IDI_LOCKICON, IDC_STATIC, 7, 59, 20, 20
-    LTEXT "Acest calculator este în uz și a fost blocat.", IDC_STATIC, 36, 61, 232, 8
-    LTEXT "<Message>", IDC_LOCKED_MESSAGE, 37, 75, 231, 26
-    LTEXT "Apăsați simultan «Ctrl»+«Alt»+«Del» pentru a debloca acest calculator.", IDC_STATIC, 36, 106, 232, 8
+    LTEXT "Acest computer este activ şi a fost blocat.", IDC_STATIC, 36, 61, 232, 8
+    LTEXT "<Mesaj>", IDC_LOCKED_MESSAGE, 37, 75, 231, 26
+    LTEXT "Apăsaţi Ctrl-Alt-Del pentru a debloca acest computer.", IDC_STATIC, 36, 106, 232, 8
 END
 
 IDD_UNLOCK DIALOGEX 0, 0, 275, 179
 STYLE NOT WS_VISIBLE | DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_BORDER | WS_CAPTION | WS_DLGFRAME | WS_POPUP
-CAPTION "Deblocare calculator"
+CAPTION "Deblocare computer"
 FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
     ICON IDI_LOCKICON, IDC_STATIC, 7, 59, 20, 20
-    LTEXT "Acest calculator este în uz și a fost blocat.", IDC_STATIC, 36, 61, 232, 8
-    LTEXT "<Message>", IDC_UNLOCK_MESSAGE, 36, 75, 232, 26
+    LTEXT "Acest computer este activ şi a fost blocat.", IDC_STATIC, 36, 61, 232, 8
+    LTEXT "<Mesaj>", IDC_LOCKED_MESSAGE, 36, 75, 232, 26
```

Comment:
Why did you change IDC_UNLOCK_MESSAGE to IDC_LOCKED_MESSAGE here?

Reply:
Sorry, I didn't pay attention. I mistakenly did it.